spring-boot/proxy/src/main/java/eu/coatrack/proxy/metrics/MetricsCounterService.java

```diff
@@ -103,7 +103,7 @@ private Metric createMetricToTransmit(TemporaryMetricsAggregation tma) {
         metricToTransmit.setRequestMethod(tma.getRequestMethod());
         metricToTransmit.setType(tma.getMetricType());
         metricToTransmit.setHttpResponseCode(tma.getHttpResponseCode());
-        metricToTransmit.setDateOfApiCall(Date.valueOf(LocalDate.now()));
+        metricToTransmit.setDateOfApiCall(new java.util.Date());
```

Comment:
Although this solves the problem that most of the metrics are stored "with the wrong day" in admin, it still does not allow admin and gateway to run in different timezones. Consider the following situation with admin in UTC timezone and gateway in UTC+2:

- it is `2022-10-14 23:59 UTC` for admin and `2022-10-15 01:59 UTC+2` for the gateway
- the gateway is continuously counting calls for the date `2022-10-15` which are now 100 calls, however, as admin is still on the previous day, it stores `100th call at 2022-10-14`
- as soon as the admin time switches to `2022-10-15 00:00 UTC` and gateway time to  `2022-10-15 02:00 UTC+2`, the gateway registers the 101st call
- when gateway now sends the information `101st call at 2022-10-15 02:00 UTC+2` then admin will adjust it to its timezone and will store in the DB  `101st call at 2022-10-15`
- this means that admin now registered `100 calls on 2022-10-14` and `101 calls on 2022-10-15` although in reality it should have been `1 call on 2022-10-15`

Summarizing the problem: this solution lets "gateway count according to its timezone" while "admin stores the count according to another timezone". We need to discuss further how to solve this issue.

Reply:
I introduced a base image for the CoatRack components whose time zone is always CET. That means, currently, the system should work when the Gateway is deployed in a CET timezone. As this is meant to be a quick fix of the problem, this solution should suffice for now. We should create a follow-up issue within which the critical occurrences of `Date` will be replaced by `ZonedDateTime` and maybe implement a policy which handles all times uniformly in UTC. Would you agree?

Reply:
In addition, the Gateway also transmits metrics and date referring to CET timezone. The SIT's worked well, too. This solution should sufficient for a "quick fix". Please, have another look.

Reply:
Update: I could drastically simplify this PR by making all components use UTC  instead of CET now.